File: Day18/Solution.java
    // declarations
    LinkedList s = new LinkedList();
    LinkedList q = new LinkedList();

    // push Character
    public void pushCharacter(char ch){
        s.add(ch);
    }

    // enqueue charactor
    public void enqueueCharacter(char ch){
        q.add(ch);
    }

    // pop Character
    public char popCharacter(){
        return (char) s.removeLast();
    }

    // dequeue Character
    public char dequeueCharacter(){
        return (char) q.removeFirst();
    }

File: Day18/Solution.py
from collections import deque

class Solution:
    def __init__(self):
        self.stack = deque()
        self.queue = deque()
    
    def pushCharacter(self,char):
        self.stack.append(char)
    
    def popCharacter(self):
        return self.stack.pop()
    
    def enqueueCharacter(self,char):
        self.queue.append(char)
    
    def dequeueCharacter(self):
        return self.queue.popleft();
